core/src/main/java/org/apache/accumulo/core/client/mapreduce/lib/partition/KeyRangePartitioner.java

```diff
@@ -29,7 +29,6 @@
  * @deprecated since 2.0.0; Use org.apache.accumulo.hadoop.mapreduce.partition instead from the
  *             accumulo-hadoop-mapreduce.jar
  */
-@Deprecated
```

Comment:
No, these are intended to be deleted in future versions. It makes no sense to undeprecate them and deduplicate. They are duplicates specifically so we can delete the ones in core in future.